pkg/solidity-utils/test/CodeDeployer.test.ts

```diff
@@ -35,7 +35,7 @@ describe('CodeDeployer', function () {
       const receipt = await (await factory.deploy(data)).wait();
       const event = expectEvent.inReceipt(receipt, 'CodeDeployed');
 
-      expect(await ethers.provider.getCode(event.args.at)).to.equal(data);
+      expect(await ethers.provider.getCode(event.args[0])).to.equal(data);
```

Comment:
What changed here?

Reply:
When I run this in node 18 `event.args` contains an array of strings, so `event.args.at` is not the actual address in the event.

Not sure why this happens; this was the only test that had this issue. I've noticed that the address in this event is not indexed; can't confirm if it's related with that.

Reply:
Update: perhaps this is a bug in ethers?

The problem was the oarameter name. `at` is a method for `Array`, and apparently it collided with the parameter name of the event. Then, the contents of `event.args` were different than the usual ones.

Replacing `at` with `destination` fixed it.